src/sidebar/util/annotation-sharing.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const serviceConfig = require('../service-config');
```

Comment:
Hmm... there is precedence for depending on `serviceConfig` in modules in `src/sidebar/util`.  I am concerned that our layering and dependency constraints are becoming unclear though. Currently all of the code in `util/` could be fairly easily reused in another application/context, or split off into a separate JS library for that purpose.

I'm not asking for any changes at the moment, just flagging it as a possible code health issue to think about.

Reply:
This is good feedback and something that did give me pause, too. May we perhaps discuss soon?